anonymizer/utils.py

```diff
@@ -0,0 +1,26 @@
+import importlib
+
+from anonymizer.base import Anonymizer
+
+
+def get_anonymizers(app_config):
+    anonymizers_module = app_config.name + '.anonymizers'
+    mod = importlib.import_module(anonymizers_module)
+
+    anonymizers = []
+    for k, v in mod.__dict__.items():
```

Comment:
can we get k/v changed to name (or `_` since you're not using it) and class/klass for clarity